Vagrantfile

```diff
@@ -342,10 +346,11 @@ def sh_install_deps(config,
     }
     cat \<\<JAVA > /etc/profile.d/java_home.sh
 if [ -z "\\\$JAVA_HOME" ]; then
-  export JAVA_HOME=$(dirname $(dirname $(readlink -f $(which java))))
+  export JAVA_HOME=/opt/jdk-11.0.2
 fi
 export SYSTEM_JAVA_HOME=\\\$JAVA_HOME
 unset JAVA_HOME
+export PATH=/opt/jdk-11.0.2/bin:\\\$PATH
```

Comment:
Why is this added to path? The packaging tests don't use java on path anymore.

Reply:
We do use Java on the path. We have lines like:

```
local java=$(which java)
```

I'll rework these tests so that they aren't using Java from the path.

Reply:
Here is one example: https://github.com/elastic/elasticsearch/blob/fef21536fa5c0a9ec10e2d715c6bf9452da35258/qa/vagrant/src/test/resources/packaging/tests/module_and_plugin_test_cases.bash#L433